ui/src/app/pages/workspace/workspace-nav-bar.tsx

```diff
@@ -191,9 +192,10 @@ export const WorkspaceNavBar = fp.flow(
 
   const navTab = (currentTab, disabled) => {
     const { name, link } = currentTab;
-    const selected = tabPath === link;
-    const hideSeparator =
-      selected || activeTabIndex === tabs.indexOf(currentTab) + 1;
+    const currentTabIndex = tabs.indexOf(currentTab);
+    const selected = activeTabIndex === currentTabIndex;
```

Comment:
equivalent to `tabPath === link` except we can now be sure that the values will be set